Code/Mantid/docs/source/algorithms/ConvolutionFitSequential-v1.rst

```diff
@@ -0,0 +1,93 @@
+
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+An algorithm designed mainly as a sequential call to PlotPeakByLogValue 
+but used within the ConvFit tab within the Indirect Analysis interface 
+to fit Convolution Functions.
+
+Workflow
+--------
+
+.. diagram:: ConvolutionFitSequential-v1_wkflw.dot
+
+
+Usage
+-----
+
+**Example - ConvolutionFitSequential**
+
+.. testcode:: ConvolutionFitSequentialExample
+
+  # Create a host workspace
+  sample = Load('irs26176_graphite002_red.nxs')
+  resolution = Load('irs26173_graphite002_red.nxs')
+
+  # Set up algorithm parameters
+  function = "name=LinearBackground,A0=0,A1=0,ties=(A0=0.000000,A1=0.0);(composite=Convolution,FixResolution=true,NumDeriv=true;name=Resolution,Workspace=__ConvFit_Resolution,WorkspaceIndex=0;((composite=ProductFunction,NumDeriv=false;name=Lorentzian,Amplitude=1,PeakCentre=0,FWHM=0.0175)))"
+  bgType = "Fixed Flat"
+  startX = -0.547608
+  endX = 0.543217
+  temp = 0
+  specMin = 0
+  specMax = sample.getNumberHistograms() - 1
+  convolve = True
+  minimizer = "Levenberg-Marquardt"
+  maxIt = 500
+  
+  # Build resolution workspace (normally done by the Convfit tab when files load)
+  AppendSpectra(InputWorkspace1=resolution.getName(), InputWorkspace2=resolution.getName(), OutputWorkspace="__ConvFit_Resolution")
+  for i in range(1, sample.getNumberHistograms()):
+    AppendSpectra(InputWorkspace1="__ConvFit_Resolution", InputWorkspace2=resolution.getName(), OutputWorkspace="__ConvFit_Resolution")  
+  
+  # Run algorithm
+  wsName = ConvolutionFitSequential(InputWorkspace=sample, Function=function ,BackgroundType=bgType, StartX=startX, EndX=endX, Temperature=temp, SpecMin=specMin, SpecMax=specMax, Convolve=convolve, Minimizer=minimizer, MaxIterations=maxIt)
+
+  # Obtain result
+  result_ws = mtd[wsName]
+  
+  print "Result has %i Spectra" %result_ws.getNumberHistograms()
+  
+  print "Amplitude 0: " + str(result_ws.readY(0)[0])
```

Comment:
Print this to a given precision to avoid the build issues.

e.g. `'%.2f' % 0.1234567`